docs/WindowsToolchain.md

```diff
@@ -0,0 +1,65 @@
+# Windows Toolchain
+
+## Nomenclature
+
+Different terms are used to reference the different portions of the tree.  These help identify the various uses for the content in the tree.
+
+* **Runtime** (`%ProgramFiles%\Swift\Runtimes\0.0.0\...`)
+
+This is the *runtime* component. It contains the libraries which are needed by the user to execute code writtenin Swift. These set of libraries are required to be redistributed with any Swift application or must be provided by the Operating System (e.g. as with macOS). This usually contains the Swift runtime, standard library. It may optionally require Foundation if it is used by the application. Only one copy of the runtime component is required, though each application may bundle its own requisite subset.
+
+* **SDK** (`%ProgramFiles%\Platforms\Windows.platform\Developer\SDKs\Windows.sdk\...`)
+
+More than one SDK may be installed at a time. There is no requirement that the platform SDK match the host. However, the host SDK is likely required as dependency on Swift within the Swift toolchain grows. The platform SDK may support multiple architectures. This content is used by the developer to build products for the specific platform for any set of architecture(s) it supports.
```

Comment:
dependency on Swift within the Swift toolchain grows -> dependency on Swift within the Swift compiler grows

That is more technically accurate and is less confusing to outsiders.

architecture(s) -> architectures

One doesn't say set of a singular, though it is technically accurate.

Reply:
I'm not sure if it is.  We are growing a dependency on Swift in the entire toolchain.  There are more components that are being added and many of them do use Swift.